src/main/java/org/apache/maven/model/jdom/etl/JDomModelETL.java

```diff
@@ -175,6 +178,15 @@ private void writePom(File pomFile) throws IOException {
       }
     }
 
+    // Remove empty properties tags for project and profile tags
```

Comment:
Cleaning up the POM should rather be configurable (add an option to the `ModelETLRequest`), and the default behavior should be not doing the clean-ups - the main goal of this library is/was to allow POM modifications retaining as much of the formats, comments etc as possible.

Reply:
Solved by https://github.com/CoreMedia/maven-jdom-parser/pull/12/commits/3e3e306c4f77780e4e575901706d9eccf64c0984